kymatio/scattering2d/tests/test_scattering2d.py

```diff
@@ -181,6 +181,17 @@ def test_FFT():
         backend.fft(y)
     assert ('must be contiguous' in record.value.args[0])
 
+    pad = backend.Pad(2 ** 0, pre_pad=False)
+
+    # -- synthesise data
+    y = torch.rand(1, 1, 8, 8)
+
+    y_pad = pad(y)                                          # pad to add complex dimension
```

Comment:
I think avoiding using pad to add a the last coordinate would be great and that right now it's mroe confusing than helping.(there is no need to pad for applying a fft)

Reply:
Agreed!